src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int BOUND_NUMBER = 50;
+    private static final int FIGURES_NUMBER = 4;
+    private static final String DEFAULT_COLOR = Color.WHITE.toString().toLowerCase();
+    private static final int DEFAULT_RADIUS = 10;
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int figureNumber = random.nextInt(FIGURES_NUMBER);
+        Figure figure;
+        switch (figureNumber) {
+            case 0:
+                figure = new Square(colorSupplier.getRandomColor(), random.nextInt(BOUND_NUMBER));
+                break;
```

Comment:
```suggestion
        switch (figureNumber) {
            case 0:
                return getRandomSquare();
```
lets add private methods to this class - this would make your code much easier to read